devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -25,6 +26,7 @@ metadata {
         capability "Health Check"
 
         fingerprint inClusters: "0000,0001,0003,000F,0020", outClusters: "0003,0019", manufacturer: "SmartThings", model: "tagv4", deviceJoinName: "SmartThings Presence Sensor"
+        fingerprint profileId: "0104", deviceId: "000C", inClusters: "0000,0001,0003,0006,0020", outClusters: "0003,0019", manufacturer: "Vision", model: "ArrivalTagv1", deviceJoinName: "Vision Zigbee Arrival Sensor"
```

Comment:
@Steven5517 I think You still need to create a separate DTH file that supports Your device only and roll back the changes that You've made in arrival-sensor-ha.groovy - that would comply with @greens suggestions.

Reply:
OK! I will create a separate DTH file that supports our device only.

Reply:
@Steven5517 you'll want to revert all of the changes to this file.